src/website/app/Router.js

```diff
@@ -70,7 +70,11 @@ export default function Router({ demos }: Props) {
                 <IconArrowBack color="currentColor" size="small" />{' '}
                 {selectedDemo.title}
               </Link>
-              <ComponentDocExample {...selectedExample} />
+              <ComponentDocExample
+                chromeless
+                parent={selectedDemo.title}
```

Comment:
`parent` is an overloaded term.  Can we be more explicit, e.g. `parentTitle`? 

Or even better, is it possible to calculate page title/path stuff here and pass them in rather than forcing `ComponentDocExample` to know about things?  You might need to standardize all these props to `pageTitle`, `pagePath` or something like `pageOptions: { title, path}`?  Let's discuss...